lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -103,6 +117,14 @@
     });
   };
 
+  function maybeJson (json) {
+    if (typeof json !== 'string') {
+      return false;
+    }
+    var cases = [ 'true', 'false', 'null', '"', '-', '[', '{' ];
+    return cases.some(function(element) { return json.split(' ')[0] === element; });
```

Comment:
Array.some is... oh, back to IE9. TIL.

This code is correct, but  you should use the second `limit` parameter to `split` to avoid doing a lot of extra work, since we'll use this a lot. We're only interested in the first result.

And, you should call `json.split(' ')[0]` just once and capture it in a variable rather than doing it on every pass through the loop. (Maybe the compiler figures this out for you, or maybe it can't.)